packages/ocamlgraph/ocamlgraph.1.8.7/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "1.2"
+maintainer: "filliatr@lri.fr"
+authors: [
+  "Sylvain Conchon"
+  "Jean-Christophe Filliâtre"
+  "Julien Signoles"
+]
+homepage: "http://ocamlgraph.lri.fr/"
+license: "GNU Library General Public License version 2.1"
+doc: ["http://ocamlgraph.lri.fr/doc"]
+dev-repo: "https://github.com/backtracking/ocamlgraph.git"
+bug-reports: "https://github.com/backtracking/ocamlgraph/issues"
+
+tags: [
+  "graph"
+  "library"
+  "algorithms"
+  "directed graph"
+  "vertice"
+  "edge"
+  "persistent"
+  "imperative"
+]
+build: [
+  ["./configure"]
+  [make]
+]
+install: [make "install-findlib"]
+remove: [["ocamlfind" "remove" "ocamlgraph"]]
+depends: ["ocamlfind"]
```

Comment:
For the next release, you may add a `{build}` attribute to the `ocamlfind` dependency. As `ocamlgraph` only depends on `ocamlfind` binary and not the associated library, it would avoid unnecessary recompilation of `ocamlgraph` whenever `ocamlfind` is updated.